pricenode/src/test/java/bisq/price/ExchangeTestBase.java

```diff
@@ -0,0 +1,74 @@
+package bisq.price;
+
+import bisq.price.spot.ExchangeRate;
+import bisq.price.spot.ExchangeRateProvider;
+
+import bisq.core.locale.CurrencyUtil;
+import bisq.core.locale.TradeCurrency;
+
+import com.google.common.collect.Sets;
+
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+import lombok.extern.slf4j.Slf4j;
+
+import static org.junit.Assert.assertTrue;
+
+@Slf4j
+public class ExchangeTestBase {
```

Comment:
Prefer `AbstractExchangeTest` for naming consistency and please mark the class as `abstract` to better signal that it is not in fact a concrete / executable test.

Reply:
Actually, prefer `AbstractExchangeRateProviderTest` for clarity, as what is being tested are indeed `ExchangeRateProvider` implementations.